### PR TITLE
feat(flipper): Add flipper relay tools

### PR DESCRIPTION
**Pull request by damassi**
### Description

This PR adds flipper support for relay, and creates the initial `system` folder in prep for moving other framework-y things into it. 

cc @artsy/mobile-platform 

### PR Checklist

- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import { LogBox } from "react-native"
 import { AppProviders } from "./AppProviders"
 import { Main } from "./navigation/Main"
+import { setupFlipper } from "./system/flipper"
+
+setupFlipper()
```
Comment (damassi) [inline src/app/App.tsx:6]: @pvinis - is this the best place for system-wide init time things? Or is there another location where boot stuff goes?

File: src/app/system/flipper.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const setupFlipper = () => {
```
Comment (damassi) [inline src/app/system/flipper.ts:1]: Other flipper plugins can go here

File: src/app/App.tsx (hunk 1 above)
Comment (pvinis) [reply src/app/App.tsx:6]: it's good here

**Review by pvinis — APPROVED** [review]
merge when you want

**Pull request merged**